chrome/content/zotero/xpcom/quickCopy.js

```diff
@@ -321,32 +331,35 @@ Zotero.QuickCopy = new function() {
 				}
 				
 				var content = {
-					text: contentType == "html" ? html : text,
+					text: format.contentType == "html" ? html : text,
 					html: copyHTML
 				};
 				
 				return content;
 			}
 			
+			// determine locale preference
+			var locale = format.locale ? format.locale : Zotero.Prefs.get('export.quickCopy.locale');
+			
```

Comment:
This is one of those places where we can't rely on using the value directly. I would suggest adding some sort of `Zotero.Styles.getLocale(style, preferredLocale)` method that would validate the locale against the style.